.github/workflows/create-release-artifacts.yaml

```diff
@@ -83,7 +83,7 @@ jobs:
       - name: Build binaries
         run: |
           export PATH="/usr/local/opt/bison/bin:$PATH"
-          make -j CXXFLAGS="-O3 -flto -DNDEBUG -mmacosx-version-min=10.9" PKG_CONFIG="pkg-config --static" PNGLDLIBS="$(pkg-config --static --libs-only-L libpng | cut -c 3-)/libpng.a $(pkg-config --static --libs-only-l libpng | sed s/-lpng[0-9]*//g)" Q=
+          make -j CXXFLAGS="-O3 -flto -DNDEBUG -mmacosx-version-min=10.9 -I include/hacks" PKG_CONFIG="pkg-config --static" PNGLDLIBS="$(pkg-config --static --libs-only-L libpng | cut -c 3-)/libpng.a $(pkg-config --static --libs-only-l libpng | sed s/-lpng[0-9]*//g)" Q=
```

Comment:
What if I just do `make` on macOS; should/will it include this directory too? (We could wrap the whole file in `#ifdef __APPLE__` and unconditionally include it in the Makefile.)

Reply:
[As I explained](https://github.com/gbdev/rgbds/pull/1280#discussion_r1436992158), only our macOS CI build sees this header, so `make` with no further args on macOS would *not* see that header (due to the lack of `-I include/hacks` being passed to the compiler).

Wrapping the header in `#ifdef __APPLE__` makes the already-unlikely breakage I mentioned there even more unlikely, so I guess we can take it.